spec/fixtures/files/datacite/ezid_item.json

```diff
@@ -58,6 +58,13 @@
             }
 	]
 	,
+	"publisher" :
```

Comment:
This comment isn't related to this particular line, but to the file as a whole.  Maybe worth renaming the file now if we're not doing EZID-related things anymore?

Reply:
File has been renamed: `spec/fixtures/files/datacite/item.json`